Python/other/Longest_Consecutive_Subsequence.py

```diff
@@ -0,0 +1,61 @@
+"""
+Given an array of positive integers. Find the length of the longest sub-sequence such that 
+elements in the subsequence are consecutive integers,
+the consecutive numbers can be in any order.
+
+Example:
+Input: {2,6,1,9,4,5,3}
+Output: 6
+
+Explanation:
+The consecutive numbers here
+are 1, 2, 3, 4, 5, 6. These 6 
+numbers form the longest consecutive
+subsquence.
+
+Example 2:
+Input: {1,9,3,10,4,20,2}
+Output: 4
+
+Explanation:
+1, 2, 3, 4 is the longest
+consecutive subsequence.
```

Comment:
REMOVE THIS PART

Reply:
But sir this will help to under stand beginners user

Reply:
@ankitaggarwal23 is this okay